api/src/main/java/org/pmiops/workbench/db/model/Participant.java

```diff
@@ -0,0 +1,71 @@
+package org.pmiops.workbench.db.model;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.pmiops.workbench.model.CohortStatus;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "participant_cohort_status")
+public class Participant {
+
+    private ParticipantKey participantKey;
+    private CohortStatus status;
+
+    @EmbeddedId
+    public ParticipantKey getParticipantKey() {
+        return participantKey;
+    }
+
+    public void setParticipantKey(ParticipantKey participantKey) {
+        this.participantKey = participantKey;
+    }
+
+    public Participant participantKey(ParticipantKey participantKey) {
+        this.participantKey = participantKey;
+        return this;
+    }
+
+    @Enumerated(EnumType.STRING)
```

Comment:
If you use a tinyint in the DB I don't think you need this.

Reply:
This is needed for sorting on status. I you don't include this it sorts by the order that the enum properties are created.